.github/workflows/deploy.yml

```diff
@@ -6,6 +6,8 @@ on:
       - 2023-04
       - 2023-07
       - 2023-10
+      - 2024-01
+      - 2024-04
```

Comment:
`2024-01` was previously added to the `2024-01` branch itself, but adding it back into `unstable` is preferred.